copier/main.py

```diff
@@ -473,7 +473,7 @@ def _render_path(self, relpath: Path) -> Optional[Path]:
                 return None
             rendered_parts.append(part)
         with suppress(IndexError):
-            if is_template:
+            if is_template and self.template.templates_suffix:
```

Comment:
With an empty suffix, the next instruction would actually empty the last rendered part (`= rendered_parts[-1][0:-0]`)